### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](b-buchkevych.github.io/layout_calendar/)
- [TEST REPORT LINK](b-buchkevych.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/day.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.calendar__day {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: #eee;
+  border: 1px solid #000;
+  @include square($cell-size);
+
```
Comment (IvanFesenko) [inline src/styles/blocks/day.scss]: Add transition for transform and background-color

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Your start day modifier didn't work
https://user-images.githubusercontent.com/61904995/228841826-41918cfd-47ba-42d8-abb7-df87a40ade45.mp4

File: src/styles/blocks/day.scss (hunk 1 above)
Comment (IvanFesenko) [reply src/styles/blocks/day.scss]: Not fixed

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Add transition

**Review by AlisaVlasova — APPROVED** [review]

**Pull request closed without merging**